journalbeat/tests/system/test_base.py

```diff
@@ -74,17 +69,7 @@ def test_start_with_selected_journal_file(self):
         )
         journalbeat_proc = self.start_beat()
 
-        required_log_snippets = [
-            # journalbeat can be started
-            "journalbeat is running",
-            # journalbeat can seek to the position defined in the cursor
-            "Reading from the beginning of the journal file",
-            # message can be read from test journal
-            "\"message\": \"thinkpad_acpi: unhandled HKEY event 0x60b0\"",
-        ]
-        for snippet in required_log_snippets:
-            self.wait_until(lambda: self.log_contains(snippet),
-                            name="Line in '{}' Journalbeat log".format(snippet))
+        self.wait_until(lambda: self.output_has(lines=23))
```

Comment:
I noticed this assertion is different to any of the former ones, and besides that, the changes in other tests are asserting that the output contains `"journalbeat is running"`, is this on purpose?